pyteal/ir/ops.py

```diff
@@ -179,6 +179,13 @@ def min_version(self) -> int:
     gitxnas             = OpType("gitxnas",             Mode.Application,                  6)
     gloadss             = OpType("gloadss",             Mode.Application,                  6)
     acct_params_get     = OpType("acct_params_get",     Mode.Application,                  6)
+    box_create          = OpType("box_create",          Mode.Application,                  7)
```

Comment:
@ahangsu A couple of remarks here - Let me know what you think?
* The current expectation is that we'll include Boxes in AVM 8 rather than AVM 7.  So, I think the box opcode versions + `MAX_TEAL_VERSION` should be changed.
* Like for AVM 7, I imagine we'll want a long-lived integration branch for AVM 8.  Can you base the PR off of https://github.com/algorand/pyteal/pull/439?
  * I created https://github.com/algorand/pyteal/pull/439 (`avm8`) from https://github.com/algorand/pyteal/pull/416 (`teal7`).  I'm hoping the proposal minimizes merge conflicts.
  * I'm imagining we'll regularly merge `master` into `teal7` and `avm8`.

Reply:
just up the version of `MAX_TEAL_VERSION` to 8, and rebased the whole branch against `avm8`